### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by ndushay**
NOTE:  please deploy this after merge is done;  give headsup in web-archiving slack channel first.

## Why was this change made?

To address vulnerability noted in github (tho it actually doesn't affect us) and gem updates to boot.

## Was the documentation (README, DevOpsDocs, API, wiki, consul, etc.) updated?

n/a

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "bootstrap": "^4.3.1",
     "jquery": "^3.4.1",
     "popper.js": "^1.15.0",
+    "serialize-javascript": "^2.1.1",
```
Comment (jcoyne) [inline package.json]: it looks like you added this dependency.  What does it do?

Comment (ndushay) [reply package.json]: I dunno;  we were getting a github alert, so i think it was a secondary dependency before, but not using latest version.

Comment (jmartin-sul) [reply package.json]: appears to be an indirect dependency of `compression-webpack-plugin`:
https://github.com/sul-dlss/was-registrar-app/blob/ebbde0ff2008e741661f2d4eac98a2d285cd690c/yarn.lock#L1733-L1743

Comment (jmartin-sul) [reply package.json]: and `terser-webpack-plugin`

https://github.com/sul-dlss/was-registrar-app/blob/ebbde0ff2008e741661f2d4eac98a2d285cd690c/yarn.lock#L6560-L6573

**Comment by ndushay** [comment]
FYI: deployed to stage and prod

**Pull request merged**